### PR TITLE
Fix tests

### PR DESCRIPTION
**Pull request by gbudau**
Fix test_http_redirect_location_301
Fix test_index_cgi_200
Fix test_index_cgi_query_string_200

Co-authored-by: Victor Dedios <victor.dedios@outlook.com>

### CODE REVIEW

File: html/web3/cgi-bin/info/hello.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env python2
```
Comment (Damrod) [inline html/web3/cgi-bin/info/hello.py:1]: In my machine, python2 is also shorthand for python2.7. Is it always the case?

File: srcs/app/CGI/CGI.cpp [hunk 2]
```diff
@@ -1,6 +1,7 @@
 #include <CGI.hpp>
 
-std::string CGI::GetExecutable_(const std::string &extension) {
+std::string CGI::GetExecutable_() {
+	const std::string extension = file_.GetPathExtension();
```
Comment (Damrod) [inline srcs/app/CGI/CGI.cpp:4]: I'm sorry we forgot about this for so long!

**Review by Damrod — APPROVED** [review]
Great! Thank you!

File: html/web3/cgi-bin/info/hello.py (hunk 1 above)
Comment (gbudau) [reply html/web3/cgi-bin/info/hello.py]: Usually, there is a symbolic link from python2 to whatever version of python2 is installed in the system.

`ls -l $(which python2)`

**Comment by gbudau** [comment]
Thank you! :tada:

**Pull request merged**